<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateExperimentDialog</class>
 <widget class="QDialog" name="CreateExperimentDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meggie - Create new experiment</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>665</width>
        <height>547</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>645</width>
        <height>540</height>
       </size>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>2</y>
         <width>641</width>
         <height>522</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Experiment base file</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="0">
              <widget class="QSplitter" name="splitter">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QLineEdit" name="FilePathLineEdit"/>
               <widget class="QPushButton" name="browseButton">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </widget>
             </item>
             <item row="2" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="showFileInfoButton">
                 <property name="text">
                  <string>Show file info</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Select raw file for the experiment:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Experiment information</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="3" column="0">
            <widget class="QTextEdit" name="textEditDescription"/>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="labelAuthor">
               <property name="text">
                <string>Experiment author:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditAuthor"/>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="labelExperimentName">
               <property name="text">
                <string>Experiment name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditExperimentName"/>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelDescription">
             <property name="text">
              <string>Experiment description:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelCreatingExperiment">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Saving experiment files (this may take a while) ...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBarCreatingExperiment">
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="cancelOkButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelOkButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateExperimentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>781</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelOkButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateExperimentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>781</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFileChooserDialog()</slot>
 </slots>
</ui>
